core/markets/storage/common.hpp

```diff
@@ -6,17 +6,8 @@
 #ifndef CPP_FILECOIN_MARKETS_STORAGE_COMMON_HPP
 #define CPP_FILECOIN_MARKETS_STORAGE_COMMON_HPP
 
-#include "primitives/types.hpp"
+#define FSM_SEND(deal, event) OUTCOME_EXCEPT(fsm_->send(deal, event))
```

Comment:
Is it safe to use here `OUTCOME_EXCEPT` which may raise an exception?

Reply:
If FSM failed, we can only report problem and stop execution.